cmsplugin_cascade/templates/cascade/bootstrap3/gallery.html

```diff
@@ -7,7 +7,7 @@
 		{% thumbnail instance.image srcset.size crop=srcset.crop upscale=srcset.upscale subject_location=srcset.subject_location as thumb %}{{ thumb.url }} {{ key }}{% if not forloop.last %},{% endif %}
 		{% endfor %}"
 	{% thumbnail instance.image instance.src.size crop=instance.src.crop upscale=instance.src.upscale subject_location=instance.src.subject_location as thumb %}
-		src="{{ thumb.url }}"{% if not instance.sizes %} width="{{ thumb.width }}" height="{{ thumb.height }}"{% endif %} />
+		src="{{ thumb.url }}"{% if not instance.sizes %} width="{{ thumb.width|stringformat:"s" }}" height="{{ thumb.height|stringformat:"s" }}"{% endif %} />
```

Comment:
but ``unlocalize`` would be the correct tag. Don't bother about that additional ``{% load i10n %}`` statement. I then would have seen immediately why you do it.